packages/core/src/core/deploy/deploy_actions.ts

```diff
@@ -106,6 +106,12 @@ export const deployActions = async (
   try {
     await deployPlan.walkAsync(async (itemId: PlanItemId): Promise<void> => {
       const item = deployPlan.getItem(itemId) as PlanItem
+      log.info('Deploy item %s', item.groupKey)
+      for (const change of item.changes()) {
+        change.detailedChanges().forEach(detailedChange => {
+          log.info('Deploy change %s (action=%s)', detailedChange.id.getFullName(), detailedChange.action)
+        })
+      }
```

Comment:
plan item has a direct accessor to the detailed changes
and you can use `wu` to work with iterables more cleanly
```suggestion
      wu(item.detailedChanges()).forEach(detailedChange => {
        log.info('Deploy change %s (action=%s)', detailedChange.id.getFullName(), detailedChange.action)
      })
```
I do wonder if we want the detailed changes log in `info`, it could be quite a few logs...

Reply:
Thanks Ori!
I switched to log.debug (also synced with Alon) and switched to `wu(item.detailedChanges()`